tests/emukit/sensitivity/test_emukit_sensitivity.py

```diff
@@ -1,12 +1,44 @@
+import mock
 import pytest
 import numpy as np
-import random
 
 from emukit.core import ContinuousParameter, ParameterSpace
-from emukit.sensitivity.monte_carlo import MonteCarloSensitivity
+from emukit.sensitivity.monte_carlo import MonteCarloSensitivity, ModelBasedMonteCarloSensitivity
 from emukit.sensitivity.test_functions import Ishigami
+from emukit.core.interfaces import IModel
 
 
+class MockModel(IModel):
+    pass
+
+@pytest.fixture
+def model():
+    model = mock.create_autospec(MockModel)
+    model.predict.return_value = (0.1*np.ones((3, 5)), 0.5*np.ones((3, 5)))
+
+    return model
+
+def test_model_based_montecarlo_sensitivity(model):
+
+    space = ParameterSpace([ContinuousParameter('x1', -np.pi, np.pi),
+                            ContinuousParameter('x2', -np.pi, np.pi),
+                            ContinuousParameter('x3', -np.pi, np.pi)])
+
+    num_mc = 1000
+    np.random.seed(0)
+    senstivity_ishigami = ModelBasedMonteCarloSensitivity(model, space)
```

Comment:
Why ishigami? I don't see the Ishigami function used

Reply:
updated.